docs/developers/get_started/getting_started_riscv_cmake.md

```diff
@@ -79,7 +79,7 @@ $ cmake -G Ninja -B ../iree-build-riscv/ \
   -DIREE_BUILD_COMPILER=OFF \
   -DIREE_ENABLE_MLIR=OFF \
   -DIREE_BUILD_SAMPLES=ON \
-  -DRISCV_TOOLCHAIN_ROOT=${RISCV_TOOLCHAIN_ROOT}
+  -DRISCV_TOOLCHAIN_ROOT=${RISCV_TOOLCHAIN_ROOT} \
```

Comment:
this should also be fixed in this doc: https://github.com/google/iree/blob/main/docs/website/docs/building-from-source/riscv.md#risc-v-64-bit-linux-target

Reply:
OK